javascriptv3/example_code/cloudwatch-events/README.md

```diff
@@ -8,14 +9,14 @@ Shows how to use the AWS SDK for JavaScript (v3) to work with Amazon CloudWatch
 <!--custom.overview.start-->
 <!--custom.overview.end-->
 
-*CloudWatch Events send system events from AWS resources to AWS Lambda functions, Amazon Simple Notification Service topics, streams in Amazon Kinesis, and other target types.*
+_CloudWatch Events send system events from AWS resources to AWS Lambda functions, Amazon Simple Notification Service topics, streams in Amazon Kinesis, and other target types._
```

Comment:
Wondering if READMEs should be included, because isn't WriteMe going to just put these right back to how they were?

Reply:
@cpyle0819 asked me about that this AM and I asked him to open a ticket to make sure the jinja template outputs pass prettier.

Reply:
@rlhagerm in addition to David's comment, I've set up my pre-commit hook to format changed files. So while writeme is in flux, my hook will still catch it.